google-cloud-clients/google-cloud-storage/src/test/java/com/google/cloud/storage/it/ITStorageTest.java

```diff
@@ -1920,6 +2041,6 @@ public void testGetServiceAccount() throws InterruptedException {
     String projectId = remoteStorageHelper.getOptions().getProjectId();
     ServiceAccount serviceAccount = storage.getServiceAccount(projectId);
     assertNotNull(serviceAccount);
-    assertEquals(SERVICE_ACCOUNT_EMAIL, serviceAccount.getEmail());
+    assertTrue(serviceAccount.getEmail().endsWith(SERVICE_ACCOUNT_EMAIL_SUFFIX));
```

Comment:
This seems to remove the check on the service account email. What service account is being used? Whatever the user has for their default application credentials?

Reply:
Currently this operation is in a strange state where it is in both `project-id@gs-project-accounts.iam.gserviceaccount.com` and service-PROJECT-NUMBER@gs-project-accounts.iam.gserviceaccount.com`. 
The update verifies the suffix exists for the Google Cloud Project id from [Specifying a Project ID](https://github.com/GoogleCloudPlatform/google-cloud-java#specifying-a-project-id).